test/integrationtests/LorisIntegrationTest.class.inc

```diff
@@ -45,10 +45,12 @@ abstract class LorisIntegrationTest extends PHPUnit_Framework_TestCase
     public function setUp()
     {
         // Set up database wrapper and config
-        $this->config = NDB_Config::singleton(__DIR__ . "/../../project/config.xml");
-        $database     = $this->config->getSetting('database');
+        $factory      = NDB_Factory::singleton();
+        $this->config = $factory->Config(CONFIG_XML);
```

Comment:
Why would you want to run tests without bootstrap? It does some of the setup, such as including `autoload.php` and `Loris_PHPUnit_Database_TestCase.php`. In fact, I am thinking of changing bootstrap not to allow running tests with 'project/config.xml'.